templates/submission/saved.tpl

```diff
@@ -20,7 +20,7 @@
             <a href="{$submissionWizardUrl}">
                 {$submission->getCurrentPublication()->getShortAuthorString()|escape}
                 —
-                {$submission->getCurrentPublication()->getLocalizedFullTitle()|escape}
+                {$submission->getCurrentPublication()->getLocalizedFullTitle(null, 'html')|unescape:'html'}
```

Comment:
As elsewhere -- I don't think the `unescape` is necessary here (and may be dangerous without filtering).

Reply:
> don't think the unescape is necessary 

Right , fixed. 

> may be dangerous without filtering

I think it is not if we keep the sanitization process . but if we remove it, then we must do it .